app/reflexes/products_reflex.rb

```diff
@@ -232,4 +241,10 @@ def product_finder(id)
   def current_id_from_element(element)
     element.dataset.current_id
   end
+
+  def image_params
```

Comment:
Is this needed ? It is not used anywhere.

Reply:
🙈 Oops, good pickup thanks!